tests/unit/test_analytics.py

```diff
@@ -1,6 +1,54 @@
 import pytest
+import mock
+import platform
+import json
 
-from dvc.analytics import Analytics
+from voluptuous import Schema, Any
+
+from dvc import analytics
+from dvc.utils.compat import str, builtin_str
+
+
+string = Any(str, builtin_str)
+
+
+@pytest.fixture
+def tmp_global_config(tmp_path):
+    """
+    Fixture to prevent modifying the actual global config
+    """
+    with mock.patch(
+        "dvc.config.Config.get_global_config_dir", return_value=str(tmp_path)
+    ):
+        yield
+
+
+def test_runtime_info(tmp_global_config):
+    schema = Schema(
+        {
+            "dvc_version": string,
+            "is_binary": bool,
+            "scm_class": Any("Git", None),
+            "user_id": string,
+            "system_info": dict,
```

Comment:
These are optional by-default, you need to use Required(), otherwise this test might not work.